Dockerfile

```diff
@@ -1,21 +1,19 @@
 #  Copyright (C) 2018-2021 LEIDOS.
-# 
+#
 #  Licensed under the Apache License, Version 2.0 (the "License"); you may not
 #  use this file except in compliance with the License. You may obtain a copy of
 #  the License at
-# 
+#
 #  http://www.apache.org/licenses/LICENSE-2.0
-# 
+#
 #  Unless required by applicable law or agreed to in writing, software
 #  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
 #  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 #  License for the specific language governing permissions and limitations under
 #  the License.
 
-FROM usdotfhwastoldev/carma-base:develop as setup
+FROM usdotfhwastoldev/carma-base:develop-arm as setup
```

Comment:
Unrelated to this PR, but we should look at [multi-platform images](https://docs.docker.com/build/building/multi-platform/) to avoid having this `*-arm` designators.

Reply:
agreed, blame @JonSmet on this namespace
this can just be a stand-in name while we work something better out. Our Docker image tags in general should be reviewed and C1T may be another decent place to trial options

Reply:
Agreed as well, I didn't realize docker images supported this but it looks like a good approach to consider.